ISeriesProvider/DB2iSeriesMappingSchema.cs

```diff
@@ -6,86 +6,160 @@
 
 namespace LinqToDB.DataProvider.DB2iSeries
 {
-	using System.Reflection;
+    using System.Linq.Expressions;
+    using System.Reflection;
 	using Extensions;
 	using Mapping;
 	using SqlQuery;
 
-	public class DB2iSeriesMappingSchema : MappingSchema

```

Comment:
Refactored MappingSchema following the DB2 -> DB2.LUW/DB2.zOS paradigm

Base instance contains the common config, then the four implementations use inheritance by constructor injection